src/gui/optionsdialog.ui

```diff
@@ -824,6 +845,9 @@
             </item>
             <item>
              <widget class="QCheckBox" name="checkPreallocateAll">
+              <property name="toolTip">
+               <string>Allocate full file sizes on disk before starting downloads, to minimize fragmentation. Only useful for HDDs.</string>
```

Comment:
"before starting downloads" is a bit misleading IMO. If a user download a orphan torrent (no seeders), the file will never be created on disk. So maybe just left that part out? I.e. "Allocate full file sizes on disk to minimize fragmentation. Only useful for HDDs."

Reply:
Sure, done.